parachain-tracer/src/tracker.rs

```diff
@@ -710,6 +720,13 @@ impl SubxtTracker {
 		std::time::Duration::from_millis(cur_ts).saturating_sub(std::time::Duration::from_millis(base_ts))
 	}
 
+	/// Returns the time for the current candidate
+	pub fn get_candidate_time(&self) -> std::time::Duration {
+		let current = self.current_relay_block_ts.unwrap_or_default();
+		let base = self.last_included_at_ts.unwrap_or(current);
```

Comment:
This will lead to the block time = 0 when there is no previous block. We should either not update the counter in this case or set it to undefined (e.g. `NaN` but I'm not sure how nan is treated by prometheus).

Reply:
We don't write the metric if there is no previous block. Same as for `para_block_time`.

Reply:
This function should return an Option to prevent metric updates when we don't have last block.